component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -517,8 +519,8 @@ private void processSMSOTPFlow(AuthenticationContext context, HttpServletRequest
                         response.sendRedirect(FrameworkUtils
                                 .appendQueryParamsStringToUrl(url, SMSOTPConstants.MOBILE_NUMBER_REGEX_PATTERN_QUERY +
                                         getEncoder().encodeToString(context.getAuthenticatorProperties()
-                                                        .get(SMSOTPConstants.MOBILE_NUMBER_REGEX)
-                                                        .getBytes()) +
+                                                .get(SMSOTPConstants.MOBILE_NUMBER_REGEX)
```

Comment:
Better if we can import these SMSOTP constants at the beginning of the class and directly call by the constant name from here.

Reply:
addressed with [31f8d49](https://github.com/wso2-extensions/identity-outbound-auth-sms-otp/pull/136/commits/31f8d49f284a10796db42765fa50021f9a6ebd07) in [https://github.com/wso2-extensions/identity-outbound-auth-sms-otp/pull/136](https://github.com/wso2-extensions/identity-outbound-auth-sms-otp/pull/136)